lib/main_common.pm

```diff
@@ -1474,7 +1474,7 @@ sub load_extra_tests_opensuse {
     return unless is_opensuse;
     loadtest "console/rabbitmq";
     loadtest "console/rails";
-    loadtest "console/machinery";
+    loadtest "console/oneclick_install" if is_tumbleweed;
```

Comment:
Our tests should always consider openSUSE Tumbleweed as the default product without a version. That means any differing behaviour should have an explicit exclude rule for the "older products" and potentially an explanation why the behaviour should differ. You can try to negate the logic check.

Reply:
This test module only works on tumbleweed so I don't see why we should do some complicated and error-prone logic like `unless (is_leap || is_jeos)` when I can just simply write what I mean.

Reply:
ok, why *does* it only work on Tumbleweed? It should work on other versions and distributions as well.

Reply:
Because the YMP file is only available for already released distributions.

Reply:
Ok but we have test for released distributions so please make sure it runs there